styles/twitch/catppuccin.user.css

```diff
@@ -611,6 +611,15 @@
       color: @text !important;
     }
 
+    .ffz-il-tooltip__container .tw-c-text-overlay {
+      background-color: fade(@base, 80%) !important;
+    }
+
+    .ffz-il-tooltip__container p,
+    .ffz-il-tooltip__container figure.ffz-i-flag {
+      color: @text;
+    }
```

Comment:
A comment here would be nice explaining what this themes, and you can using nesting here to keep it organized.